react/AccordionItem/AccordionItem.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import styles from './AccordionItem.less';
+import { ChevronIcon } from '../';
+
+type Props = {
+  children: React$Node,
+  title: React$Node,
+  id: string | number,
+  expanded?: boolean,
+  onClick?: Function,
+  className?: string
+};
+
+type State = {
+  expanded?: boolean
```

Comment:
This is not optional on `state`,
```suggestion
  expanded: boolean
```